modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/agg/Accumulators.java

```diff
@@ -118,6 +117,7 @@ private Supplier<Accumulator> avgFactory(AggregateCall call) {
     private Supplier<Accumulator> sumFactory(AggregateCall call) {
         switch (call.type.getSqlTypeName()) {
             case BIGINT:
+                return () -> new Sum(new LongSumEmptyIsZero());
```

Comment:
What if accumulator value overflows? Can we detect this?
Does SQL standard defines aggregate return types? or it's implementation defined?

Reply:
At the moment COUNT returns BIGINT. It MAP/REDUCE version of it should also produce the same type.

Reply:
Ok, you are right.
Same questions. What about "SUM" aggregate below?